docs/Module-Review/Y1S1/MH1805 Calculus.md

```diff
@@ -0,0 +1,27 @@
+---
+id: mh1805-calculus
+slug: mh1805-calculus
+sidebar_position: 3
+title: MH1805 Calculus
+description: Introductory calculus course.
+---
+
+## Course Summary
+
+This is a course that covers a good deal of concepts relating to calculus, from basics like limits and first principles, to concepts that are most likely new, like Taylor series approximations and big-o notation.
+
+## Workload
+
+Workload is quite heavy. Being a 4AU course, there is naturally more content to cover, and for most students a lot of this content is likely either new or very rigorous. There are weekly lectures, tutorials, as well as midterms and a final exam.
+
+## Projects
+
+No projects.
+
+## Things to take note of
+
+The content covered in this course is quite challenging, and there is a lot of new mathematical notation and ideas that are introduced. Students are taught to think and prove mathematical ideas rigorously. This can be a challenging learning curve, but is also one that makes subsequent math modules (like MH2100 Calculus III) more manageable.
```

Comment:
Possible to make this a hyperlink to the Calc 3 page?

Reply:
Just updated